lesson14/main.js

```diff
@@ -82,6 +88,7 @@ closeModal.addEventListener("click", () => {
 });
 
 requestBtn.addEventListener("click", () => {
+  console.log(getNumbers());
   init();
   modal.style.display = "none";
```

Comment:
Here is 

`modal.classList.add("hidden");`

like that couldn't to be handled ?

Reply:
Oh!!!! Thank you for telling me, I tried on some way and totally forgot to fix that ;(
Now I fixed.